### PR TITLE
parametrise bedrock region

### PR DESCRIPTION
**Pull request by georschi**
*Issue #, if available:*
Bedrock region was hardcoded

*Description of changes:*
Updates to allow for Bedrock region to be configured in the config file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: setup/configs.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 [
   {
-    "STACK_PREFIX": "",
-    "BEDROCK_ENDPOINT": "https://bedrock-runtime.{}.amazonaws.com",
+    "STACK_PREFIX": "{ENTER_PREFERRED_STACK_NAME_PREFIX}",
+    "BEDROCK_ENDPOINT": "https://bedrock-runtime.{ENTER_PREFERRED_BEDROCK_REGION}.amazonaws.com",
```
Comment (brunopistone) [inline setup/configs.json]: Bedrock Region is taken from the stack. This change will break the .replace in the file `app.py`

File: setup/app.py [hunk 2]
```diff
@@ -139,7 +140,8 @@ def build(self):
             memory=512,
             timeout=900,
             environment={
-                "BEDROCK_URL": "https://bedrock-runtime.us-east-1.amazonaws.com",
+                "BEDROCK_URL": self.bedrock_endpoint_url,
+                "BEDROCK_REGION": self.bedrock_region
```
Comment (brunopistone) [inline setup/app.py]: Can you please take as region `self.region`?

**Review by brunopistone — CHANGES_REQUESTED** [review]

**Comment by brunopistone** [comment]
Hey @georschi , the solution is currently using the region where you are deploying the resources (see `app.py`). The changes in config.json are breaking the deployment. Regarding the environments variables for the lambda, this is a good point.
Can you make the following changes:
1. `app.py`: provide a parameter `"BEDROCK_REGION": self.region` for the lambda functions "list_foundation_models" and "invoke_model"

**Comment by georschi** [comment]
Thanks @brunopistone , that should be good now

**Review by brunopistone — APPROVED** [review]
Approved

**Pull request merged**